app/lib/shared/integrity.dart

```diff
@@ -40,6 +40,11 @@ class IntegrityChecker {
   final DatastoreDB _db;
   final int _concurrency;
 
+  static const _knownUnmappedFields = {
+    'Package.isWithheld',
+    'Package.withheldReason',
+  };
+  final _unmappedFields = <String>{};
```

Comment:
Just to make sure, do we create a new IntegrityChecker for each run?

Reply:
Yes, this is a one-off object on each run.